apps/web/next.config.mjs

```diff
@@ -12,10 +12,13 @@ const nextConfig = {
   output: 'standalone',
   transpilePackages: INTERNAL_PACKAGES,
   serverExternalPackages: [
+    '@latitude-data/sdk',
+    '@latitude-data/promptl',
+    '@latitude-data/compiler',
```

Comment:
Why this?

Reply:
adding telemetry started to collide with nextjs transpilation so i just decided to make all compiled packages external since there's not really a point in having go through next's build pipeline